--- fml.py	2013-04-23 20:56:28 +0900
+++ fml.py.new	2013-10-03 08:36:38 +0900
@@ -12,7 +12,7 @@
 
 def download_deps(mcp_path):
     ret = True
-    for lib in ['argo-3.2-src.jar','guava-14.0-rc3.jar','asm-4.1.tar.gz','asm-debug-all-4.1.jar','bcprov-debug-jdk15on-148.jar','bcprov-jdk15on-148-src.zip','guava-14.0-rc3-sources.jar', 'scala-library.jar']:
+    for lib in ['argo-3.2-src.jar','guava-14.0-rc3.jar','asm-4.1.tar.gz','asm-debug-all-4.1.jar','bcprov-debug-jdk15on-148.jar','bcprov-jdk15on-148-src.zip','guava-14.0-rc3-sources.jar', 'scala-library.jar.stash']:
         libF = os.path.join(mcp_path, 'lib')
         if not os.path.isdir(libF):
             os.makedirs(libF)
@@ -26,7 +26,9 @@
             except:
                 print 'Download %s failed, download manually from http://files.minecraftforge.net/fmllibs/%s or http://files.minecraftforge.net/fmllibs/fml_libs_dev15.zip and place in mcp/lib' % (lib, lib)
                 ret = False
-    
+    from_f = os.path.normpath(os.path.join(libF, 'scala-library.jar.stash'))
+    to_f   = os.path.normpath(os.path.join(libF, 'scala-library.jar'))
+    os.rename(from_f, to_f)
     return ret
 
 def config_get_section(config, section):
